src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2841,6 +2841,209 @@ Note: `aparse()` is case-insensitive.
+                <td>`["test", [...]]`</td>
+              </tr>
+              <tr>
+                <td>'valueB.nestedValue2'</td>
+                <td>Returns value at key</td>
+                <td>`100`</td>
+              </tr>
+              <tr>
+                <td>'valueC[0]'</td>
+                <td>Returns list member</td>
+                <td>`{"id": 1…}`</td>
+              </tr>
+              <tr>
+                <td>'valueC[0,2]'</td>
+                <td>Returns first and third members</td>
+                <td>`[{"id": 1…}, {"id": 3…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[0:2]'</td>
+                <td>Returns first through 2 members, exclusive</td>
+                <td>`[{"id": 1…}, {"id": 2…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[:2]'</td>
+                <td>Returns the first 2 members, exclusive</td>
+                <td>`[{"id": 1…}, {"id": 2…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[:-2]'</td>
+                <td>Returns everything except the last two members</td>
+                <td>`[{"id": 1…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[1:]'</td>
+                <td>Returns the second to the last members</td>
+                <td>`[{"id": 2…}, {"id": 3…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[-1:]'</td>
+                <td>Returns the last member</td>
+                <td>`[{"id": 3…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[*]'</td>
+                <td>Returns all members</td>
+                <td>`[{"id": 1…}, {"id": 2…},{"id": 3…}]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[*].id'</td>
+                <td>Returns key of members</td>
+                <td>`[1, 2, 3]`</td>
+              </tr>
+              <tr>
+                <td>'valueC[*]["label", other]'</td>
+                <td>Returns keys of members</td>
+                <td>`[["A", 7],…]`</td>
+              </tr>
+            </tbody>
+          </table>
+    </Collapser>
+    <br />
+
+    Examples:
+
+    <CollapserGroup>
+      <Collapser id="basic-jparse-example" title="Basic Example">
+        The following query parses the JSON string within the `jsonString` attribute.
+
+        ```sql
+          WITH '{"user": {"name": "John", "id": 5}}' as jsonString SELECT jparse(jsonString)
+        ```
+
+        This query will return the deserialized JSON string:
+        ```
+          {"user":{"name":"John","id":5}}
+        ```
+      </Collapser>
+      <Collapser id="parse-specific-value-from-log-message" title="Parse a specific value from a log message">
+        A common problem is having rich and structured data hiding with a Log message. You can leverage `aparse()` and `jparse()` to trim away noise and find specific values within a log message.
```

Comment:
...rich and structured data within a Log message...?

Reply:
Fixed.